BlackNectar/BlackNectar/StoresTableViewController.swift

```diff
@@ -50,6 +51,7 @@ class StoresTableViewController: UITableViewController, FilterDelegate, UIGestur
     }
     
     var stores: [Store] = []
+    let mailComposeViewController = MFMailComposeViewController()
```

Comment:
You should not need to create and store the mail controller as a variable. Just create and use it when you need it.